tests/tracking/fluent/test_fluent.py

```diff
@@ -247,35 +229,6 @@ def test_get_experiment_by_id():
         assert experiment.experiment_id == exp_id
 
 
-def test_get_experiment_by_id_with_is_in_databricks_job():
```

Comment:
Can we add a test case verifying that `get_experiment_by_id` delegates to `DefaultExperimentProvider` as expected? This is necessary to replace the coverage that we're losing by removing this test case.